recipes/libverto/all/conanfile.py

```diff
@@ -106,62 +111,60 @@ def requirements(self):
-    def _configure_autotools(self):
-        if self._autotools:
-            return self._autotools
-        self._autotools = AutoToolsBuildEnvironment(self, win_bash=tools.os_info.is_windows)
-        self._autotools.libs = []
+        self.build_requires("pkgconf/1.9.3")
+        self.build_requires("libtool/2.4.7")
+        if self._settings_build.os == "Windows":
+            self.win_bash = True
+            if not self.conf.get("tools.microsoft.bash:path", check_type=str):
+                self.tool_requires("msys2/cci.latest")
+
+    def generate(self):
+        env = VirtualBuildEnv(self)
+        env.generate()
+        tc = AutotoolsToolchain(self)
         yes_no = lambda v: "yes" if v else "no"
         yes_no_builtin = lambda v: {"external": "yes", "False": "no", "builtin": "builtin"}[str(v)]
-        args = [
-            "--enable-shared={}".format(yes_no(self.options.shared)),
-            "--enable-static={}".format(yes_no(not self.options.shared)),
-            "--with-pthread={}".format(yes_no(self.options.get_safe("pthread", False))),
-            "--with-glib={}".format(yes_no_builtin(self.options.with_glib)),
-            "--with-libev={}".format(yes_no_builtin(self.options.with_libev)),
-            "--with-libevent={}".format(yes_no_builtin(self.options.with_libevent)),
-            "--with-tevent={}".format(yes_no_builtin(self.options.with_tevent)),
-        ]
-        self._autotools.configure(args=args, configure_dir=self._source_subfolder)
-        return self._autotools
+        tc.configure_args.extend([
+            f"--with-pthread={yes_no(self.options.get_safe('pthread'))}",
+            f"--with-glib={yes_no_builtin(self.options.with_glib)}",
+            f"--with-libev={yes_no_builtin(self.options.with_libev)}",
+            f"--with-libevent={yes_no_builtin(self.options.with_libevent)}",
+            f"--with-tevent={yes_no_builtin(self.options.with_tevent)}",
+            ])
+
+        env = tc.environment()
+        # if is_msvc(self):
+        #     # FIXME: Use the conf once https://github.com/conan-io/conan-center-index/pull/12898 is merged
+        #     # env.define("AR", f"{unix_path(self, self.conf.get('tools.automake:ar-lib'))}")
+        #     [version_major, version_minor, _] = self.dependencies.direct_build['automake'].ref.version.split(".", 2)
+        #     automake_version = f"{version_major}.{version_minor}"
+        #     ar_wrapper = unix_path(self, os.path.join(self.dependencies.direct_build['automake'].cpp_info.resdirs[0], f"automake-{automake_version}", "ar-lib"))
+        #     env.define("CC", "{} cl -nologo".format(unix_path(self.dependencies.direct_build["automake"].compile))) 
+        #     env.define("CXX", "{} cl -nologo".format(unix_path(self.deps_user_info["automake"].compile))) 
+        #     env.define("LD", "link -nologo") 
+        #     env.define("AR", f"{ar_wrapper} \"lib -nologo\"")
+
+        tc.generate()
+        deps = AutotoolsDeps(self)
+        deps.generate()
```

Comment:
Is it really required?

Reply:
This question is important because `AutotoolsDeps` can break "all shared" configuration if VirtualRunEnv is not injected in build scope.
So either:
- upstream configure.ac is able to find all dependencies with pkgconfig and AutotoolsDeps should be removed
- or it must be kept and you have to inject VirtualRunEnv in build scope

Reply:
It isn't required. I removed it